jsk_2016_01_baxter_apc/launch/segmentation_in_bin.launch

```diff
@@ -28,46 +24,26 @@
   <node name="sib_topic_synchronizer"
     pkg="jsk_apc2016_common" type="sib_topic_synchronizer"
     output="screen">
-    <remap from="~input" to="$(arg INPUT_POINTS)"/>
+    <remap from="~input" to="$(arg INPUT_CLOUD)"/>
     <remap from="~input/info" to="$(arg INPUT_INFO)"/>
     <remap from="~input/image" to="$(arg INPUT_IMAGE)"/>
   </node>
 
-  <node name="segmentation_in_bin"
-        pkg="jsk_2016_01_baxter_apc" 
-        type="rbo_segmentation_in_bin_node.py" 
-        output="screen">
-        <remap from="~input" to="/sib_topic_synchronizer/output"/>
-    <remap from="~input/bin_info_array" to="$(arg INPUT_BIN_ARRAY)"/>
-    <remap from="~target_bin_name" to="$(arg INPUT_TARGET)"/>
-    <param name="trained_pkl_path" value="$(find jsk_apc2016_common)/data/trained_segmenter.pkl" />
-  </node>
-
-   
-  <arg name="MANAGER" value="segmentation_in_bin_manager" />
-  <node name="$(arg MANAGER)"
-        pkg="nodelet" type="nodelet" args="manager" />
-
-  <node name="sib_mask_image_to_pi"
-        pkg="nodelet" type="nodelet"
-        args="load jsk_pcl/MaskImageToPointIndices $(arg MANAGER)">
-    <remap from="~input" to="/segmentation_in_bin/target_mask"/>
-  </node>
-
-  <node name="sib_extract_pi"
-        pkg="nodelet" type="nodelet"
-        args="load jsk_pcl/ExtractIndices $(arg MANAGER)">
-    <remap from="~input" to="$(arg INPUT_POINTS)" />
-    <remap from="~indices" to="/sib_mask_image_to_pi/output" />
-    <rosparam>
-      max_queue_size: 100
-      approximate_sync: true
-    </rosparam>
-  </node>
-
-  <node name="sib_centroid_publisher"
-        pkg="nodelet" type="nodelet"
-        args="load jsk_pcl/CentroidPublisher $(arg MANAGER)">
-        <remap from="~input" to="/sib_extract_pi/output"/>
-  </node>
+  <include file="$(find jsk_2016_01_baxter_apc)/launch/include/left_hand_segmentation_in_bin.launch">
+    <arg name="INPUT_CLOUD" value="$(arg INPUT_CLOUD)" />
+    <arg name="MANAGER" value="$(arg MANAGER)" />
+    <arg name="JSON" value="$(arg json)" />
```

Comment:
Can you make left_hand_segmentation_in_bin.launch and right_hand_segmentation_in_bin.launch into one file?
I think you can by using an argument that indicates left or right.

If you think that it is too complicating, then that is fine too.